lib/document-helpers.ts

```diff
@@ -143,3 +143,9 @@ export {
   getTotalVolume,
   getTotalActiveNodes,
 };
+
+export const getTotalMinutes = async () => {
+  const response = await fetch("https://explorer.livepeer.org/api/usage");
```

Comment:
Is this endpoint new and where is it served from?

Reply:
This is the same data that powers the charts on the explorer. What's new is the API endpoint that now exposes this data.

Reply:
https://github.com/livepeer/livepeerjs/pull/956/files

Reply:
Nice 👍